src/components/mgt-person-card/mgt-person-card.scss

```diff
@@ -0,0 +1,139 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+@import '../../../node_modules/office-ui-fabric-core/dist/sass/References';
+@import '../../styles/shared-styles.scss';
+
+$avatar-size-s: var(--avatar-size-s, 24px);
```

Comment:
These variables seem like a copy and paste - do we need them here?

Reply:
e14cd3d

Reply:
forgot one of the imports, readded here:

554db18